src/manifest.json

```diff
@@ -238,11 +238,13 @@
         }
     ],
     "optional_permissions": [
+        "https://*/*",
```

Comment:
This still worries me since it degrades the extension's security when gitlab is enabled

Reply:
The jetbrains toolbox extension is doing the same with custome git instances.

https://github.com/JetBrains/toolbox-browser-extension/blob/master/manifest.json

How about adding a extra button for customer websites. This would be a solution not only for gitlab.
We can easily describe the security implections there.

Reply:
Sorry for the delay, I missed this comment the other day.
As long as this change doesn't broaden the permissions for users who don't use this feature, I'm fine with it. Is this the case?

Reply:
Jup. 

Only when you activate the custom gitlab option this permission will be activated.